lib/utilities/monitoringHandler.js

```diff
@@ -0,0 +1,65 @@
+const errors = require('arsenal');
+const client = require('prom-client');
+
+function writeResponse(res, error, log, results, cb) {
+    let statusCode = 200;
+    if (error) {
+        if (Number.isInteger(error.code)) {
+            statusCode = error.code;
+        } else {
+            statusCode = 500;
+        }
+    }
+    res.writeHead(statusCode, { 'Content-Type': 'application/json' });
+    res.write(JSON.stringify(results));
+    res.end(() => {
+        cb(error, results);
+    });
+}
+
+
+function routeHandler(req, res, log, cb) {
+    if (req.method !== 'GET') {
+        return cb(errors.BadRequest, []);
+    }
+    client.collectDefaultMetrics();
```

Comment:
This should be only called once at application startup, not per-request?

Reply:
So what do you suggest the timeout value to be. I thought we can control it better with the server controlling the timeout.

Reply:
Also, existing intervals are automatically cleared when calling collectDefaultMetrics.

Reply:
I'd just set the interval to 5s or so, we won't be using a higher sampling rate, and not call it for every server request.